packages/cli/src/commands/generate/scaffold/templates/components/NameForm.tsx.template

```diff
@@ -27,21 +37,39 @@ const ${singularPascalName}Form = (props) => {
           titleClassName="rw-form-error-title"
           listClassName="rw-form-error-list"
         />
-<% editableColumns.forEach(column => { %>
+      <% editableColumns.forEach(column => { %>
```

Comment:
I imagine the could be done as "code" dynamically instead of rendering out each enum value, but this is for clarity.